_resources/port1.0/group/legacysupport-1.1.tcl

```diff
@@ -175,7 +175,8 @@ proc legacysupport::add_legacysupport {} {
         set ls_cache_cppflags "[legacysupport::get_cpp_flags]"
 
         # Flags for using MP libcxx
-        if { [option legacysupport.use_mp_libcxx] } {
+        global configure.cxx_stdlib
```

Comment:
Please move this global to the top of the `proc` and place with the other global statements.

Reply:
@cjones051073 I moved it where it is now following @catap advice: https://github.com/macports/macports-ports/pull/16076#discussion_r973782693
I am okay with either, but there should be some consensus :)

Reply:
If you look at other places globals are used, in other portgroups or base itself we generally always define globals at the start of procs.

Reply:
@cjones051073 Ok, gonna move it.